content/release/10.0.md

```diff
@@ -0,0 +1,143 @@
+
+E.g. to change the AddPath/ORF (Outbound Route Filtering) capability's flags, a session reset is not needed if the dynamic capability is enabled between the peers.
+
+### SRv6 encapsulation source address feature
+
+Configure the source address of the outer encapsulating IPv6 header.
+
+More details are [here](https://docs.frrouting.org/en/latest/zebra.html#clicmd-encapsulation).
+
+### OSPFv3 Point-To-Multipoint mode
+
+Add an ability to set the network type to point-to-multipoint for an interface.
+
+More details are [here](https://docs.frrouting.org/en/latest/ospf6d.html#ospf6-point-to-point-and-point-to-multipoint-operation).
+
+### Other significant changes
+
+bgpd
+- Add `clear bgp capabilities` command to resend some dynamic capabilities [link](https://docs.frrouting.org/en/latest/bgp.html#clicmd-clear-bgp-ipv4-ipv6-unicast-PEER-capabilities)
+- Add `debug bgp updates detail` command [link](https://docs.frrouting.org/en/latest/bgp.html#clicmd-debug-bgp-updates-detail)
+- Add `debug bgp updates <in|out> <X.Y.Z.W> prefix-list <NAME>` command [link](https://docs.frrouting.org/en/latest/bgp.html#clicmd-debug-bgp-updates-in-out-A.B.C.D-X-X-X-X-WORD-prefix-list-WORD)
+- Add `neighbor capability fqdn` command [link](https://docs.frrouting.org/en/latest/bgp.html#clicmd-neighbor-PEER-capability-fqdn)
+- Add `redistribute table-direct` support [link](https://docs.frrouting.org/en/latest/bgp.html#clicmd-redistribute-table-table-direct-1-65535-metric-0-4294967295-route-map-WORD)
+- Fix `match ip address ...` + `match evpn ...` commands for EVPN
+- Remove aggregated (summary-only) suppressed routes from EVPN
+
+mgmtd
+- Implement full XPATH 1.0 predicate functionality
+- Output `staticd` configuration from `mgmtd`
+
+ospfd
+- Fix crash in OSPF TE parsing
+
+ospf6d
+- Advertise local addresses with la bit
+- Set loopback interface cost to 0
+- Let the user override interface cost for a loopback
+
+pathd
+- Add dynamic candidate path metric [computed] keyword [link](https://docs.frrouting.org/en/latest/pathd.html#clicmd-metric-bound-METRIC-VALUE-required-computed)
+- Add `no msd` command in the `pcc` context
+- Add `no pcep` command
+
+vtysh
+- Send interface commands to mgmtd
+
+watchfrr
+- Extend the ignore option to the daemon being killed
+
+zebrad
+- Add `mpls label dynamic-block` command [link](https://docs.frrouting.org/en/latest/zebra.html#clicmd-mpls-label-dynamic-block-16-1048575-16-1048575)
+- Add JSON support to `show debugging label-table` [link](https://docs.frrouting.org/en/latest/zebra.html#clicmd-show-debugging-label-table)
+- Add zebra to `mgmtd` oper-state
+- Allow longer prefix matches for the next hops
+- Push all configured IP addresses when the interface comes up
+- Remove static ARP entries on interface-down events
+- Support to listen `teamd` netlink message as bond type
+- Fix crash when macvlan link-interface is in another netns
+
+### Breaking changes
```

Comment:
Can you please move at the top of release overview. 

Also, another item to add:

* Writing configuration works only with the unified configuration file `frr.conf`. I.e, writing to per-daemon config files is no longer supported.